python/lsst/ts/ess/ess_csc.py

```diff
@@ -0,0 +1,68 @@
+__all__ = ["EssCsc"]
+
+import logging
+import pathlib
+
+from lsst.ts import salobj
+
+
+class EssCsc(salobj.ConfigurableCsc):
+    """Upper level Commandable SAL Component for the Environmental Sensors Support.
+
+    Parameters
+    ----------
+    config_dir : `string`
+        The configuration directory
+    initial_state : `salobj.State`
+        The initial state of the CSC
+    simulation_mode : `int`
+        Simulation mode (1) or not (0)
+    """
+
+    def __init__(
+        self, config_dir=None, initial_state=salobj.State.STANDBY, simulation_mode=0,
+    ):
+        schema_path = (
+            pathlib.Path(__file__).resolve().parents[4].joinpath("schema", "ess.yaml")
+        )
+        self.config = None
+        self._config_dir = config_dir
+        self.log = logging.getLogger("EssCsc")
```

Comment:
The superclass (`salobj.ConfigurableCsc`) already has a `self.log` attribute which even sends the log messages through SAL.